idd/Energy+.idd.in

```diff
@@ -16057,6 +16064,372 @@ SurfaceProperty:OtherSideConditionsModel,
+  A13, \field Custom Block 6 Material Name
+       \type object-list
+       \object-list MaterialName
+  N25, \field Custom Block 6 Depth
+       \units m
+       \type real
+       \minimum> 0.0
+  N26, \field Custom Block 6 X Position
+       \units m
+       \type real
+  N27, \field Custom Block 6 Z Position
+       \units m
+       \type real
+  A14, \field Custom Block 7 Material Name
+       \type object-list
+       \object-list MaterialName
+  N16, \field Custom Block 7 Depth
+       \units m
+       \type real
+       \minimum> 0.0
+  N29, \field Custom Block 7 X Position
+       \units m
+       \type real
+  N30, \field Custom Block 7 Z Position
+       \units m
+       \type real
+  A15, \field Custom Block 8 Material Name
+       \type object-list
+       \object-list MaterialName
+  N31, \field Custom Block 8 Depth
+       \units m
+       \type real
+       \minimum> 0.0
+  N32, \field Custom Block 8 X Position
+       \units m
+       \type real
+  N33, \field Custom Block 8 Z Position
+       \units m
+       \type real
+  A16, \field Custom Block 9 Material Name
+       \type object-list
+       \object-list MaterialName
+  N34, \field Custom Block 9 Depth
+       \units m
+       \type real
+       \minimum> 0.0
+  N35, \field Custom Block 9 X Position
+       \units m
+       \type real
+  N36, \field Custom Block 9 Z Position
+       \units m
+       \type real
+  A17, \field Custom Block 10 Material Name
+       \type object-list
+       \object-list MaterialName
+  N37, \field Custom Block 10 Depth
+       \units m
+       \type real
+  N38, \field Custom Block 10 X Position
+       \units m
+       \type real
+       \minimum> 0.0
+  N39; \field Custom Block 10 Z Position
+       \units m
+       \type real
+
+Foundation:Kiva:Settings,
+       \memo Settings applied across all Kiva foundation calculations.
+       \memo Object is not required. If not defined, defaults will applied.
+  N1,  \field Soil Conductivity
+       \type real
+       \default 1.73
+       \units W/m-K
+       \minimum> 0.0
+  N2,  \field Soil Density
+       \type real
+       \default 1842
+       \units kg/m3
+       \minimum> 0.0
+  N3,  \field Soil Specific Heat
+       \type real
+       \default 419
+       \units J/kg-K
+       \minimum> 0.0
+  N4,  \field Ground Solar Absorptivity
+       \type real
+       \default 0.9
+       \units dimensionless
+       \minimum 0.0
+       \maximum 1.0
+  N5,  \field Ground Thermal Absorptivity
+       \type real
+       \default 0.9
+       \units dimensionless
+       \minimum 0.0
+       \maximum 1.0
+  N6,  \field Ground Surface Roughness
+       \type real
+       \default 0.03
+       \units m
+       \minimum> 0.0
+  N7,  \field Far-Field Width
+       \type real
+       \default 40
+       \units m
+       \minimum> 0.0
+  A1,  \field Deep-Ground Boundary Condition
+       \type choice
+       \key ZeroFlux
+       \key GroundWater
+       \key Autocalculate
+       \default Autocalculate
```

Comment:
Oh - this is a key choice "Autocalculate", not an autocalculatable field.  This would be the only place in the entire IDD with "\key Autocalculate".  I guess there's no rule against using it, but "autocalculate" does have special meaning in numeric fields.  Is there a different word that would convey the same intent?